DS/DS6/UniqueCharacter.java

```diff
@@ -0,0 +1,84 @@
+import java.io.*;
+import java.util.*;
+//class to convert in concordance
+public class UniqueCharacter {
+	static private HashMap<String,String> uniqueMap=new HashMap<String,String>();
+	HashSet<Character> characterSet;
+	HashMap<Character, Integer> concordanceMap;
+	public void readCharacter(String inputString) {//method to read character from input string
+		characterSet =new HashSet<Character>();
```

Comment:
Why do you need a set for this requirement, you can do it only by usig HashMap as key of a hashmap itself is unique.